ow_sim_tests/test/test_camera_set_exposure.test

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<launch>
+
+    <arg name="gzclient" default="true" />
+
+    <include file="$(find ow)/launch/europa_terminator_workspace.launch">
+        <arg name="rqt_gui" value="true" />
```

Comment:
It just occurred to me, for the purpose of server-side automated testing it will be important to make rqt_gui a command line argument.
```suggestion
    <arg name="rqt_gui" defaullt="true" />
    
    <include file="$(find ow)/launch/europa_terminator_workspace.launch">
        <arg name="rqt_gui" value="$(arg rqt_gui)" />
```
Please make the same change for `test_camera_capture.test`.

Reply:
Can I know why this change is important?

Reply:
Code test servers will run these tests "headless" meaning there is no display connected to the server, and even if there were a display connected there will be no human to observe it, so spending computation overhead on rendering and GUI logic would be a waste. Wasting computation is one thing, but many program will just crash if they attempt to use a display that is not there, so allowing these rostests to run entirely headless is a must to enable them to run on test servers.